drivers/flash/nrf_qspi_nor.c

```diff
@@ -83,8 +83,12 @@ struct qspi_cmd {
  *  device is enabled
```

Comment:
Could you add some doxygen description for the ready variable in the single-thread case? I am not sure how this could be done "conditionally". Alternatively, you could move the documentation inline , in the stucture

Reply:
I hadn't even noticed the documentation hidden up there in a block.  Moved field doc into the structure and added the new field.